mmap/src/main/c/Makefile

```diff
@@ -54,19 +48,10 @@ OBJS = \
 $(REALLIBNAME): $(OBJS)
 	$(CC) $(LD_FLAGS) -o $@ $(OBJS)
 
-build: $(REALLIBNAME) $(if $(OS:linux=),,$(if $(ARCH:x86_64=),,build32))
+build: $(REALLIBNAME) $(if $(OS:linux=),,$(if $(ARCH:x86_64=),,))
 
-build32:
-	$(MAKE) ARCH=i386 build
-
-install: build $(if $(OS:linux=),,$(if $(ARCH:x86_64=),,install32))
+install: build $(if $(OS:linux=),,$(if $(ARCH:x86_64=),,))
 	cp $(REALLIBNAME) $(DEST_DIR)/$(subst -$(ARCH),,$(REALLIBNAME))
 
-install32:
```

Comment:
All of these ifs can be removed now.

`$(if $(OS:linux=),,$(if $(ARCH:x86_64=),,))`
means
`If it's linux, output nothing`
`otherwise if it's x86_64, output nothing`
`otherwise output nothing`

Reply:
Removed